internal/cueutils/convert/to_testapi/from_sarif.cue

```diff
@@ -140,7 +140,14 @@ _findings: list.Sort(list.Concat([ for run in input.runs {
                     }
                 }
             }]
+            if result.suppressions != _|_ {
+            	if result.suppressions[0].justification != _|_ {
+								suppression: {
+									kind: "other" // TODO: Hard-coded as this does not map well to kind|state fields from sarif
```

Comment:
Another idea, it might be nice to specify `kind` and `justification` as

```suggestion
								suppression: kind: "ignored"
```

and then below, in an `if len(result.suppressions) > 0`:

```
							suppression: justification: result.suppressions[0].justification
```